src/main/java/de/vitagroup/num/domain/Roles.java

```diff
@@ -51,4 +51,12 @@ public static boolean isAllowedToSet(String roleToSet, List<String> callersRoles
   public static boolean hasRole(String role, Jwt principal) {
     return extractRoles(principal).contains(role);
   }
+
+  public static boolean isSuperAdmin(List<String> roles) {
```

Comment:
If that was not checked before, isn't that a bugfix then? Would be good to comment it in the story.

Reply:
This is just a convenience method as I restructured some code, no bug here.